api/version/version.go

```diff
@@ -55,24 +55,24 @@ func Current() string {
 	return Get().SemVer()
 }
 
-// GetInterface returns the overall codebase version. It's for detecting
+// Get returns the overall codebase version. It's for detecting
 // what code a binary was built from.
 // These variables typically come from -ldflags settings and in
 // their absence fallback to the settings in pkg/version/base.go.
 func Get() Info {
 	var (
 		gitMajor string
 		gitMinor string
-		gitPatch string = "0"
+		gitPatch = "0"
 		gitPre   string
 		gitMeta  string
 	)
 
 	v, err := semver.NewVersion(gitVersion)
 	if err == nil {
-		gitMajor = strconv.Itoa(int(v.Major()))
-		gitMinor = strconv.Itoa(int(v.Minor()))
-		gitPatch = strconv.Itoa(int(v.Patch()))
+		gitMajor = strconv.FormatUint(v.Major(), 10)
+		gitMinor = strconv.FormatUint(v.Minor(), 10)
+		gitPatch = strconv.FormatUint(v.Patch(), 10)
```

Comment:
Ok, but you have to specify more irrelevant info (10), so why not directly using the method describing what you want to do.

Reply:
It was warning about overflow casting because it was casting uint64 into and int. And FormaUint is type safe with the 10 at the end. I'll create a constant for the ten.